Source/Debugger.pas

```diff
@@ -687,7 +727,7 @@ procedure TDebugger.AddBreakPoints(
         MapLineNumber := AMapScanner.LineNumberByIndex[LineIndex];
 
         // RINGN:Segment 2 are .itext (ICODE).
-        if (MapLineNumber.Segment in [1, 2]) then
+        if (MapLineNumber.Segment in [1]) then

```

Comment:
What is the reason for removing segment 2?

Reply:
![image](https://github.com/user-attachments/assets/63e58a75-9126-4045-99f2-a877403b1b38)
I made this change because the last .end (for example, on line 163) was being included in the .map as valid code with a breakpoint. However, since I work with interfaces, CodeCoverage was not counting this line in the report, which prevented the coverage from ever reaching 100%. I couldn’t reproduce this situation again because we changed the structure of our projects .
I will go back to the code and we can evaluate this point again in the future.

Reply:
Clear, I understand your issue. Please make a separate PR for this if you know the complete fix.